aws/resource_aws_organizations_organization.go

```diff
@@ -400,6 +400,12 @@ func resourceAwsOrganizationsOrganizationUpdate(d *schema.ResourceData, meta int
 		}
 	}
 
+	if d.HasChange("feature_set") {
+		if _, err := conn.EnableAllFeatures(&organizations.EnableAllFeaturesInput{}); err != nil {
+			return fmt.Errorf("error changing feature_set to ALL")
```

Comment:
small change here: we should also return the error received with the organization id e.g.
```suggestion
			return fmt.Errorf("error enabling all features in Organization (%s): %w", d.Id(), err)
```